### ISSUE TITLE
Have you tried some lightweight net in ImageNet replacing VGG-16?

### ISSUE THREAD

**Issue by shenguquanxin** [zwx8981/DBCNN-PyTorch#10]
Good work,Thanks for sharing!!! And I have some questiones:
1 、Have you tried some lightweight net in ImageNet replacing VGG-16? such as mobilenet、shufflenet. And how about the result comparing with vgg-16
2、what the function of the "net_params_best.pkl" in content "fc_model" 
3、Do you plan to add distortion kind,such as "sharpness",in order to assess whether the enhance image is good or bad
Again,So excellent you work is!!!
You can repay with Chinese if possible.

**Comment by zwx8981** [comment]
@shenguquanxin  Thank you for your question.
1、I haven't try other pre-trained network. However, since our method is model-agnostic, you can simply replace the VGG-16 with other network. Just keep on point in mind, because we use bilinear pooling as the image representation, the number of channel of the last convolutional layer of your network should not be too large.（否则最后的维度会extremely high）
2、We use a two-stage fine-tuning strategy. In the first stage, only the newly-initialized fc layer is updated (freezing all previous layers)，and the best weights in this stage will be stored in the fc_model folder (named net_params_best.pkl). In the second stage, we continue to fine-tune all network based on the best weight saved in the first stage.
3、Actually, an concurrent work that extending our method for the BIQA of contrasted-altered scenarios has been done. You may read 'No-reference quality assessment for contrast-altered images using an end-to-end deep framework'.

**Comment by shenguquanxin** [comment]
@zwx8981 太感谢了！
我已经复现完您的模型（不包括S-cnn），基本可以达到论文水平。而且对自然图像评价比较准确。
我测试了一些故意虚化的图像，就不是那么准确了，这可能也会目前IQA模型都没有解决的问题（测试了DeepIQA、RankIQA）。
对您的工作和IQA非常感兴趣，持续关注。
